tests/finetune_test.py

```diff
@@ -0,0 +1,2 @@
+__author__ = "lucaspavanelli"
```

Comment:
Along with tests, please edit the usage docs! It is one of the most important parts of this PR.